setup.sh

```diff
@@ -300,9 +300,12 @@ function synchronize_files()
   if command_exists "zsh"; then
     # Add tabcompletion to zshrc
     if [[ -f "$HOME/.zshrc" || -L "$HOME/.zshrc" ]]; then
-      echo "# Enable bash completion for zsh" >> "$HOME/.zshrc"
-      echo "autoload bashcompinit && bashcompinit" >> "$HOME/.zshrc"
+      local zshcomp=$'# Enable bash completion for zsh\n'
```

Comment:
couldnt this be just `local zshcomp=$'# Enable bash completion for zsh\n autoload bashcompinit && bashcompinit'` ?

Reply:
Yes, but it takes up more than 80 characters, so I split it.

Reply:
I see, good call, but I wonder if there is a better way of splitting this to be less than 80 characters. Does it work if you introduce a real break line instead of \n?

Reply:
I'm not sure what you mean with "real line break", can you elaborate?

Reply:
literally pressing enter instead of using \n

Reply:
I think it's possible, but it would mess with the indentation, unless I'm missing something. It would look something like:
```
if command_exists "zsh"; then
  if [[ -f "$HOME/.zshrc" || -L "$HOME/.zshrc" ]]; then
    local zshcomp='# Enable bash completion for zsh
autoload bashcompinit && bashcompinit'
    safe_append "$zshcomp" "$HOME/.zshrc"
  fi
fi
```
If you have a better way of doing it let me know.